source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -8612,6 +8687,14 @@ export class AdaptiveCard extends ContainerWithActions {
     get hasVisibleSeparator(): boolean {
         return false;
     }
+
+    public registerMouseEnterCallback(callback: any) {
+        this._mouseEnterCallbacks.push(callback);
```

Comment:
If the goal is to reveal all inputs when the mouse enters the card, isn't there a simpler solution, maybe even a CSS only one? Namely add a `:hover` style to the `ac-adaptiveCard` class that adds the appropriate style to anything that has the ac-input style under the card?